app/build.gradle

```diff
@@ -44,11 +44,17 @@ dependencies {
   implementation 'eu.mhutti1.utils.storage:android-storage-devices:0.6.2'
 
   // Android Support
+  //noinspection GradleDependency
   implementation "com.android.support:appcompat-v7:$supportLibraryVersion"
+  //noinspection GradleDependency
   implementation "com.android.support:support-v13:$supportLibraryVersion"
+  //noinspection GradleDependency
   implementation "com.android.support:support-v4:$supportLibraryVersion"
+  //noinspection GradleDependency
   implementation "com.android.support:design:$supportLibraryVersion"
+  //noinspection GradleDependency
```

Comment:
Are you sure we don't want this to be inspected ?

Reply:
yes